package.json

```diff
@@ -39,34 +34,35 @@
   "engines": {
     "node": ">=7"
   },
+  "types": "index.d.ts",
   "dependencies": {
     "dynamodb-data-types": "^3.0.1",
     "dynamodb-update-expression": "^0.1.21",
     "dynamodb-wrapper": "^1.4.1",
-    "ramda": "^0.27.0"
+    "ramda": "^0.27.1"
   },
   "peerDependencies": {
     "aws-sdk": "2.x"
   },
   "devDependencies": {
-    "@commitlint/config-conventional": "^9.1.1",
-    "commitlint": "^9.1.0",
+    "@commitlint/config-conventional": "^11.0.0",
+    "commitlint": "^11.0.0",
```

Comment:
Could you change this to the new package?

Reply:
Done.